pkg/liqonet/ipam.go

```diff
@@ -251,7 +266,7 @@ func (liqoIPAM *IPAM) mapNetwork(network string) (string, error) {
 
 /* Helper function to get a mask from a net.IPNet */
 func getMask(network string) uint8 {
-	stringMask := network[len(network)-2:]
+	stringMask := strings.Split(network, "/")[1]
```

Comment:
can we avoid manipulating strings in order to get the mask length? i would use the "net" package to achieve the same result. something like
```
	_, network, err := net.ParseCIDR(network)
	if err != nil{
		return err
	}
	ones := network.Mask.Size())
	return ones

Reply:
Good idea